tests/unit/builder/test_brat_builder.py

```diff
@@ -50,7 +57,7 @@
             "id": ["A1", "A2"],
             "type": ["factuality", "statement"],
             "target": ["T1", "R1"],
-            "value": ["actual", "true"],
+            "value": ["actual", None],
```

Comment:
This was to cover the [line 25](https://github.com/ArneBinder/pie-datasets/blob/c042d672154850aad984855f50fc4b9edb9ce163/src/pie_datasets/builders/brat.py#L25) in brat.py. Also, it cannot be `None`, then maybe we also don't need `None` as [default value](https://github.com/ArneBinder/pie-datasets/blob/c042d672154850aad984855f50fc4b9edb9ce163/src/pie_datasets/builders/brat.py#L20) in BratAttribute ?

Reply:
but it should be triggered when it is `"true"` and not `None`, or did I miss sth?

Reply:
having `"true"` as default is a bit complicated, because in fact it should be `True`, but that is in conflict with the other values which are of type `str`.

Reply:
How can we satisfy the line `if self.value is None` in resolve method for BratAttribute ?

Reply:
Suggested changes implemented in [0126de5](https://github.com/ArneBinder/pie-datasets/pull/143/commits/0126de5e5d7246d838ad78f3197e579213198000) and tested with `test_generate_document`